data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/task/result/ImportTaskResult.java

```diff
@@ -0,0 +1,25 @@
+package com.scalar.db.dataloader.core.dataimport.task.result;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder
+@Value
+@JsonDeserialize(builder = ImportTaskResult.ImportTaskResultBuilder.class)
+public class ImportTaskResult {
```

Comment:
Honestly speaking, it's hard to me to get the relationship of these classes and how to use them from this PR. It would be great if there is a simple diagram or something to show the relationship.

Reply:
Sorry, my previous comment might be unclear. I just read the design document again. The definition of the following terms are still unclear to me, so adding comments about the definitions of xxxxx would be enough:
- `batch` (I guess `ImportTransactionBatch` is related)
- `task` (I guess `ImportTaskAction` and `ImportTaskResult` are related)

Reply:
@inv-jishnu Thanks for the explanation! I think the information will be very helpful for future code reviewers and maintainers. Could you add the description to the design doc or as source code comments?

Reply:
@komamitsu san,
I will check with Yves and add these in the design doc.
Thank you.